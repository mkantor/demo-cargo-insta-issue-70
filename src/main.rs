fn main() {}

#[test]
fn repro() {
    use insta::{assert_snapshot, Settings};
    use std::path::Path;
    use std::time::SystemTime;

    let non_constant_value = SystemTime::now()
        .duration_since(SystemTime::UNIX_EPOCH)
        .unwrap()
        .as_secs()
        .to_string();

    let mut settings = Settings::new();
    let snapshot_path = Path::new(env!("CARGO_MANIFEST_DIR")).join("snapshots");
    settings.set_snapshot_path(snapshot_path);
    settings.bind(|| assert_snapshot!(non_constant_value));
}
