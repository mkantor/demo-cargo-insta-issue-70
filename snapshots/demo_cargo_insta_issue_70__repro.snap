---
source: src/main.rs
expression: non_constant_value
---
1598984338
